### ISSUE TITLE
Error from handlebars

### ISSUE THREAD

**Issue by cartercarlson** [OpenZeppelin/solidity-docgen#406]
I am unable to run `npx hardhat docgen` after a fresh `npm install solidity-docgen`.  The error I'm getting is:
```
Error: Missing helper: "joinLines"
    at Object.<anonymous> (/home/carter/Documents/metokens/metokens-core/node_modules/handlebars/lib/handlebars/helpers/helper-missing.js:10:13)
    at Object.wrapper (/home/carter/Documents/metokens/metokens-core/node_modules/handlebars/lib/handlebars/internal/wrapHelper.js:10:19)
    at eval (eval at createFunctionContext (/home/carter/Documents/metokens/metokens-core/node_modules/handlebars/lib/handlebars/compiler/javascript-compiler.js:265:23), <anonymous>:15:112)
    at prog (/home/carter/Documents/metokens/metokens-core/node_modules/handlebars/lib/handlebars/runtime.js:333:12)
    at execIteration (/home/carter/Documents/metokens/metokens-core/node_modules/handlebars/lib/handlebars/helpers/each.js:50:9)
    at Object.<anonymous> (/home/carter/Documents/metokens/metokens-core/node_modules/handlebars/lib/handlebars/helpers/each.js:63:13)
    at Object.wrapper (/home/carter/Documents/metokens/metokens-core/node_modules/handlebars/lib/handlebars/internal/wrapHelper.js:10:19)
    at eval (eval at createFunctionContext (/home/carter/Documents/metokens/metokens-core/node_modules/handlebars/lib/handlebars/compiler/javascript-compiler.js:265:23), <anonymous>:11:49)
    at Object.prog [as fn] (/home/carter/Documents/metokens/metokens-core/node_modules/handlebars/lib/handlebars/runtime.js:333:12)
    at Object.<anonymous> (/home/carter/Documents/metokens/metokens-core/node_modules/handlebars/lib/handlebars/helpers/if.js:19:22) {
  description: undefined,
  fileName: undefined,
  lineNumber: undefined,
  endLineNumber: undefined,
  number: undefined
}
```

I've gone through the handlebars source code and haven't found anything that references to `joinLines` and have solidity v0.8.9.

**Comment by cartercarlson** [comment]
I have changed no settings on the hardhat `config`.

**Comment by frangio** [comment]
Thanks for the issue. Should be fixed in b130fd95225802f15f34c9cd00d161b1f0987ec9. Please try again.

**Comment by cartercarlson** [comment]
New error @frangio :
```
carter@blade:~/Documents/metokens/metokens-core$ npx hardhat docgen
Nothing to compile
No need to generate any newer typings.
An unexpected error occurred:

Error: Circular dependency detected: aliased imports not supported
    at run (/home/carter/Documents/metokens/metokens-core/node_modules/solidity-docgen/src/utils/scope.ts:16:15)
    at run (/home/carter/Documents/metokens/metokens-core/node_modules/solidity-docgen/src/utils/scope.ts:36:29)
    at run (/home/carter/Documents/metokens/metokens-core/node_modules/solidity-docgen/src/utils/scope.ts:36:29)
    at run (/home/carter/Documents/metokens/metokens-core/node_modules/solidity-docgen/src/utils/scope.ts:36:29)
    at getContractsInScope (/home/carter/Documents/metokens/metokens-core/node_modules/solidity-docgen/src/utils/scope.ts:7:10)
    at parseNatspec (/home/carter/Documents/metokens/metokens-core/node_modules/solidity-docgen/src/utils/natspec.ts:103:49)
    at natspec (/home/carter/Documents/metokens/metokens-core/node_modules/solidity-docgen/src/common/properties.ts:18:22)
    at /home/carter/Documents/metokens/metokens-core/node_modules/solidity-docgen/src/site.ts:103:53
    at Object.get (/home/carter/Documents/metokens/metokens-core/node_modules/solidity-docgen/src/utils/memoized-getter.ts:16:19)
    at lookupProperty (/home/carter/Documents/metokens/metokens-core/node_modules/handlebars/lib/handlebars/runtime.js:130:26)
```
